Codeinwp/neve
# Refactor/repeater

### Summary


Refactors the repeater control by implementing it in react.

### Will affect visual aspect of the product

NO

### Screenshots 
![Screenshot from 2021-07-16 11-32-14](https://user-images.githubusercontent.com/39873395/125918495-c83b5a0d-4e1d-47f2-9349-6b096cdaf653.png)
![Screenshot from 2021-07-16 11-33-11](https://user-images.githubusercontent.com/39873395/125918612-538204aa-3129-4452-be12-ff0a7d3fb7b0.png)
![Screenshot from 2021-07-16 11-33-32](https://user-images.githubusercontent.com/39873395/125918634-1b70f9ce-dfb7-4cd5-b1f6-ac1e9077ee66.png)


### Test instructions


- Apply the changes from the PR in neve-pro-addon repository from branch refactor/repeater since the repeater control is used there (https://github.com/Codeinwp/neve-pro-addon/pull/1463)
- The NEVE_VERSION constant must be >= 3.0.0 for changes to be visible.
- The controls that use the repeater and should be tested are: Social Icons, Contact and Sharing Icons from single blog page



Closes #2652.
